code/controllers/configuration/entries/general.dm

```diff
@@ -82,6 +82,16 @@
 	config_entry_value = 600
 	min_val = 0
 
+/datum/config_entry/number/vote_autotransfer_initial //length of time before the first autotransfer vote is called (deciseconds, default 2 hours)
+	config_entry_value = 72000
+	integer = FALSE
```

Comment:
The value is in deciseconds and byond ticks are 1 ds minimum. I don't understand why it should be a float.